sim-cli/src/main.rs

```diff
@@ -42,8 +48,12 @@ fn deserialize_f64_greater_than_zero(x: String) -> Result<f64, String> {
 #[derive(Parser)]
 #[command(version, about)]
 struct Cli {
+    /// Path to a directory containing simulation files, and where simulation results will be stored
+    #[clap(long, short, env = "SIM_LN_DATA_DIR", default_value = DEFAULT_DATA_DIR)]
```

Comment:
Why does this have an `env` option and `sim_file` doesn't?

Reply:
no other option is configurable through `env`. removed

Reply:
Oh, I thought this was there because it was being used by the docker setup

Reply:
Using the [flag](https://github.com/bitcoin-dev-project/sim-ln/pull/159/commits/e01198b204e2e29a009ef398c4a84663bc482fb3#diff-4f5cabe26761257a4d685a6edc7a43e0fe0f78762f50eeb48530f2bd3b3ee7caR8) so we're good 👍

Reply:
I _think_ that we could update the docker scripts to do things more efficiently (use docker env vars to set config options rather than mapping to flag values) _but_ we'd need to make other config options `env` as well to do that consistently so good to leave it out for now.